src/translations/en.json

```diff
@@ -5194,8 +5194,8 @@
           "primary_color": "Primary color",
           "accent_color": "Accent color",
           "reset": "Reset",
-          "backend-selected": "Use backend preferred theme",
-          "default": "Default"
+          "use_default": "Use default theme",
+          "home_assistant": "Home Assistant"
```

Comment:
This should not be a translation in this case.

Instead, we should hard-code this, as it is a name that isn't translatable.

../Frenck

Reply:
Oh right 🤦‍♂️ I don't know why I did this 😅